test/e2e/atlas/operator_helper_test.go

```diff
@@ -167,7 +158,7 @@ func (oh *operatorHelper) stopOperator() {
 		oh.t.Errorf("unable to retrieve operator deployment: %v", err)
 	}
 
-	deployment.Spec.Replicas = toptr.MakePtr(int32(0))
+	deployment.Spec.Replicas = pointer.Get(0)
```

Comment:
This has to be `int32` not `int`

Reply:
@gssbzn fyi ^ probably that's why the other method was used.

Reply:
pointer.Get is a generic
```
pointer.Get[int32](0)
```

Reply:
fixed